vargrid.go

```diff
@@ -261,6 +266,28 @@ func (config *VarGridConfig) LoadPopMort() (*Population, PopIndices, *MortalityR
 	return &Population{tree: pop}, PopIndices(popIndex), &MortalityRates{tree: mort}, nil
 }
 
+// Constants stores values that do not vary over grid cells
+type PopConstants map[string]float64
```

Comment:
Would govaluate allow us to define these constants as OutputVariables instead of in a separate file? Is the problem then that all the constants would get written to the output shapefile? If this is the problem, we could get around it by only writing OutputVariables to the shapefile when the first letter in the variable name is capitalized.